shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/PoolReap.hs

```diff
@@ -41,12 +42,13 @@ instance STS (POOLREAP crypto) where
   type Environment (POOLREAP crypto) = PParams
   type BaseM (POOLREAP crypto) = ShelleyBase
   data PredicateFailure (POOLREAP crypto)
-    = FailurePOOLREAP
     deriving (Show, Eq, Generic)
   initialRules = [pure $ PoolreapState emptyUTxOState emptyAccount emptyDState emptyPState]
   transitionRules = [poolReapTransition]
 
-instance NoUnexpectedThunks (PredicateFailure (POOLREAP crypto))
+instance NoUnexpectedThunks (PredicateFailure (POOLREAP crypto)) where
```

Comment:
Do you plan to wait for https://github.com/input-output-hk/cardano-prelude/pull/101 to be merged so you can just use `deriving`?

Reply:
Since you voiced support for 101: yes, I'll update this one to use it and wait for it

cc: @JaredCorduan @uroboros

Reply:
Fixed, thanks.